joomla/joomla-cms
# [4.0] Update composer versions

### Summary of Changes
This does a simple update of our composer dependencies. It doesn't try and make any major version bumps in order to keep this as a more simple risk free PR

### Testing Instructions
There's no specific things to test. Both the symfony and session dependencies are things that will are "under the hood" changes and will be used on most pages in different ways. So just navigate around a bit and try to see if anything breaks.

### Documentation Changes Required
None
